libc/docs/platform_support.rst

```diff
@@ -0,0 +1,20 @@
+Platform Support
+================
+
+Development is currently mostly focused on Linux.  MacOS and Windows has
+partial support, but has bitrot and isn't being tested continuously.
+
+LLVM-libc is currently being integrated into Android and Fuchsia operating
+systems via `overlay mode <overlay_mode.html>`__.
+
+For Linux, we support kernel versions as listed on
+`kernel.org <https://kernel.org/>`_, including ``longterm`` (not past EOL
+date), ``stable``, and ``mainline`` versions. We actively adopt new features
+from ``linux-next``.
+
+For Windows, we plan to support products within their lifecycle. Please refer to 
+`Search Product and Services Lifecycle Information <https://learn.microsoft.com/en-us/lifecycle/products/?products=windows>`_ for more information.
+
```

Comment:
Maybe add a line for the GPU build that links to https://libc.llvm.org/gpu/.

Reply:
28abaef62273